packages/core-components/src/layout/SignInPage/providers.tsx

```diff
@@ -95,6 +96,7 @@ export const useSignInProviders = (
         IdentityApiSignOutProxy.from({
           identityApi,
           signOut: async () => {
+            localStorage.removeItem(LAST_SEEN_ONLINE_STORAGE_KEY);
```

Comment:
Would it be safe to skip this? It'd be best to avoid this cross-component communication. There are also other implementations of the sign-in page for which this will not be called.